test/integration/targets/win_pester/defaults/main.yml

```diff
@@ -1,2 +1,5 @@
 ---
 test_win_pester_path: C:\ansible\win_pester
+parameter:
```

Comment:
this should be inline in the test; it's not being shared, so make it obvious to someone looking at the test what the test is passing

Reply:
changed